marconi/src/Marconi/Indexers.hs

```diff
@@ -138,16 +137,41 @@ datumWorker Coordinator{_barrier} ch path = Datum.open path (Datum.Depth 2160) >
               offset <- findIndex (any (\(s, _) -> s < slot)) events
               Ix.rewind offset index
 
+utxoHotStoreWorker :: IndexerHotStore -> TargetAddresses -> Worker
```

Comment:
What does that HotStore term mean in this context?

Reply:
@koslambrou  there are now 2 utxoWorkers, functions,  in the Marconi.Indexers module:

1. utxoWorker, no change in this function. It stores the indexers on disk.   My mental model for this worker is `ColdStore`, however I didn't want to change the function name as it seemed unnecessary at this point. 
2. utxoHotStoreWorker , a new function that stores to TVar, cache, so I refer to it as HotStore.

The terms helped me keep the two concepts separated. 
I think a better solution might be to just have one utxoWorker, and pass in a function that does the storage, but it makes the function signature very verbose, and complicated looking, however, using ReaderT will help with that problem. I discussed using the `mtl` library in marconi with @raduom and it is an option that we might do post the big refactor effort

Reply:
Cool. Thanks for clarification :)